ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.html

```diff
@@ -10,7 +10,7 @@
           <div style="padding-right: 64px;">
             <p class="ig-about-title" i18n>{{  this.interoperabilityGuidelineItem?.title ?? '-' }}</p>
             <p class="ig-about-subtitle" i18n>
-              {{ ( this.interoperabilityGuidelineItem?.eosc_guideline_type ?? '-' | guidelineTypePipe ) }}
+              {{ ( this.interoperabilityGuidelineItem?.eosc_guideline_type ?? '-' | filterPipe: DICTIONARY_TYPE_FOR_PIPE.GUIDELINE_TYPE ) }}
```

Comment:
I think that pipe should possible handle also the part `... ?? '-'`. It would be shorter and more concise

Reply:
refactor after merge